tests/test_config.py

```diff
@@ -24,6 +24,9 @@ def test_enabled_star():
 
             [[python]]
             python_path = /some/path
+
+            [[xpidl]]
+            header_bucket = /somewhere
```

Comment:
Ah, probably makes sense to say path. My mental imagery is that there's a bucket and we toss headers in.